ompi/mca/bcol/basesmuma/bcol_basesmuma_allreduce.c

```diff
@@ -268,7 +269,7 @@ int bcol_basesmuma_allreduce_intra_fanin_fanout(bcol_function_args_t *input_args
 {
     /* local variables */
     mca_bcol_basesmuma_module_t *bcol_module = (mca_bcol_basesmuma_module_t *) c_input_args->bcol_module;
-    int buff_idx = buff_idx = input_args->src_desc->buffer_index;
```

Comment:
@shamisp @manjugv There was no deep/secret meaning for assigning `buff_idx = buff_idx = some_other_value`, right?